### DISCUSSION TITLE
Can swap sd card to new device?

### DISCUSSION THREAD

**Discussion by ghackett** [OnionUI/Onion#536] [Q&A] [answered]
I've got a 2nd device coming in the mail just in case something happens to mine. Assuming it has the correct firmware installed, would I be able to just drop in my SD card with onionOS already installed/setup or is something written to the internal chip during onionOS install, where i'd need to go through that process again?

**Comment by Olywa** [comment, marked as answer]
Yup, you can just swap the card as long as you've upgraded the new device fw to 0419 (April) version as it will likely arrive with the Jan fw. 
Nothing form OnionOS is written to the device itself.
